docs/konnectors.md

```diff
@@ -320,18 +320,40 @@ On the host create the file /etc/rkt/net.d/10-containers.conf
 
 and run your container with the "--net=bridge" option. That way, a new interface is available in the container and gives you access to the host.
 
+## Konnector install and run details
+
+### Install
+
+The konnectors will be installed in the .cozy_konnectors directory (not in VFS ?) using npm install
+(with the ```--ignore-scripts``` option). This command will give the possibility to install konnectors
+from npm, github, gitlab, bitbucket, tar.gz url. The stack will have to check that the given package
+does not use a "file:" path.
+
+In the .cozy_konnectors directory, a ```npm ls -depth 0``` will be able to give the list of installed
+konnectors (npm outdated can also be used) or yarn equivalent.
```

Comment:
Updates was supposed to be a reinstall of the same package, using npm or yarn or git does not change anything for this.

If you talk about update notification, I see that npm our yarn are not of any help if packages are git repositories or anything not an npm package. The only way would be to check version of each package.json for each source which will be a pain.